fluentd-gcp-image/Dockerfile

```diff
@@ -20,7 +20,7 @@
 # scope and that the Logging API has been enabled for the project
 # in the Google Developer Console.
 
-FROM gcr.io/google-containers/debian-base-amd64:0.3.2
+FROM gcr.io/google-containers/debian-base-amd64:0.4.1
```

Comment:
Can we migrate this to distroless?

Reply:
We need bash to run script. distroless has no version that includes bash interpreter https://github.com/GoogleContainerTools/distroless/blob/master/base/README.md

Reply:
We don't need to run the script. Essentially, all that script is doing is running a binary. We might as well just run the binary...

Reply:
Ok, I was thinking about "fluentd-gcp-scaler" which is implemented in bash.
I don't know enough about ruby to give definite answer, by my understanding is that ruby is interpreted language and distroless doesn't support it. I think this would be a major effort for this image which is being superseded by stackdriver agent

Reply:
Ack. We should probably put a deprecation note in the README and point people to stackdriver-logging-agent instead.